packages/cryptoplease/lib/features/outgoing_split_key_payments/src/bl/oskp_verifier.dart

```diff
@@ -22,7 +22,7 @@ class OSKPVerifier {
     _repoSubscription = _repository.watchWithReadyLinks().listen((payments) {
       for (final payment in payments) {
         void onSuccess(String txId) {
-          final newStatus = OSKPStatus.success(txId: txId);
+          final newStatus = OSKPStatus.success(withdrawTxId: txId);
           _repository.save(payment.copyWith(status: newStatus));
```

Comment:
I think we're doing that since `txId` on OSKP is the "send to escrow" tx id. The `withdrawTxId` is the one we track once the receiver use the links. We're not handling this "cancellation" case, though. There's a PR to allow user cancel OSKP (#626), but I think we also do not handle this match there.